### ISSUE TITLE
Frontend JS error due to translation

### ISSUE THREAD

**Issue by rrrmity** [verbb/comments#138]
### Description
On a plain installation of CraftCMS the plugin is installed and rendered using`{{ craft.comments.render(entry.id) }}`.
Commenting works fine, but if one clicks on reply, edit, delete etc. an JS error occurs:
```
[Error] TypeError: undefined is not an object (evaluating 'Comments.translations.hasOwnProperty')
	t (comments.js:1:3886)
	reply (comments.js:1:6709)
	reply
```
This can be nailed down to `return Comments.translations.hasOwnProperty(e)?Comments.translations[e]:""`, where `Comments.translations` is null. That is because in the init `t.translations` which is used to set Comments.translations is null (set here `Comments.translations=t.translations`)

We could see this problem on different systems and installations, but are not sure, where the problems originate from. Does anyone have a clue/hint what to do?

Our workaround for now is to change
`return Comments.translations.hasOwnProperty(e)?Comments.translations[e]:""`
to
`return ""` and everything is working as expected (the frontend is english only), but this of course is just a dirty hack to make it work on dev.

### Steps to reproduce

1. Install plugin
2. render comments with `{{ craft.comments.render(entry.id) }}`
3. press reply, delete, etc.

### Additional info

- Plugin version: 1.1.9
- Craft version: 3.3.0.1

**Comment by dreamseeker** [comment]
Hi, I occured same issue.

I thought it was caused by not setting `translations` property when creating a `Comments` instance.

```html
<script src="pathTo/cpresources/XXXXX/js/comments.js?v=NNNNNNN"></script>
<script>
  new Comments.Instance("#cc-w-NNN", {
      "template": ...,
      "variables": {...},
      "id": ...,
      "element": {...},
      "commentsQuery": {...},
      "settings": {...}
  });
</script>
```

So I tried to make the following changes to https://github.com/verbb/comments/blob/craft-3/src/services/CommentsService.php#L58-L81

```php
// Prepare variables to pass to templates - important to include route params
$routeParams = Craft::$app->getUrlManager()->getRouteParams();

$variables = array_merge($routeParams, [
    'id' => $id,
    'element' => $element,
    'commentsQuery' => $query,
    'settings' => $settings,
    'baseUrl' => UrlHelper::actionUrl(),
    'csrfTokenName' => Craft::$app->getConfig()->getGeneral()->csrfTokenName,
    'csrfToken' => Craft::$app->getRequest()->getCsrfToken(),
    'recaptchaEnabled' => (bool)$settings->recaptchaEnabled,
    'recaptchaKey' => $settings->recaptchaKey,
    'translations' => [
        'reply' => Craft::t('comments', 'Reply'),
        'close' => Craft::t('comments', 'Close'),
        'edit' => Craft::t('comments', 'Edit'),
        'save' => Craft::t('comments', 'Save'),
        'delete-confirm' => Craft::t('comments', 'Are you sure you want to delete this comment?'),
    ]
]);
```

After that, It looks fine.

**Comment by engram-design** [comment]
Sorry about this, fixed in [1.1.10](https://github.com/verbb/comments/releases/tag/1.1.10)
